060_Distributed_Search/20_Scan_and_scroll.asciidoc

```diff
@@ -17,53 +14,46 @@ are made to the index after the initial search request has been made. It does
 this by keeping the old data files around, so that it can preserve its ``view''
 on what the index looked like at the time it started.
 
---
-
-`scan`::
-
 The costly part of deep pagination is the global sorting of results, but if we
 disable sorting, then we can return all documents quite cheaply. To do this, we
-use the `scan` search type.((("scan search type"))) Scan instructs Elasticsearch to do no sorting, but
-to just return the next batch of results from every shard that still has
-results to return.
+sort by `_doc`. This instructs Elasticsearch just return the next batch of 
+results from every shard that still has results to return.
 
-To use _scan-and-scroll_, we execute a search((("scan-and-scroll"))) request setting `search_type` to((("search_type", "scan and scroll")))
-`scan`, and passing a `scroll` parameter telling Elasticsearch how long it
-should keep the scroll open:
+To scroll through results, we execute a search request and set the `scroll` value to
+the length of time we want to keep the scroll open.  
```

Comment:
Re-reading this section, it seems the timeout explanation is only partially explained by us.  I think we should move the later timeout explanation up here.  And we should explain why the timeout is needed in the first place.  I'm thinking maybe:

> To scroll through results, we execute a search request and set the scroll value to the length of time we want to keep the scroll "open".  The scroll expiry time is refreshed every time we run a scroll request, so it needs to give us only enough time to process the current batch of results, not all of the documents that match the query.  The timeout is important; resources are being used to hold the scroll window open and we want to free those as soon as they are no longer needed.  A timeout allows the resources to automatically be freed after a small period of inactivity.
>
> GET /old_index/_search?scroll=1m <1>
> [...]

Not overly tied to the language used, just the gestalt.  Feel free to munge or disagree :)